packaging/network-monitor/miden-network-monitor.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Miden network monitor
+Wants=network-online.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=exec
+EnvironmentFile=/lib/systemd/system/miden-network-monitor.env
+ExecStart=/usr/bin/miden-network-monitor
+WorkingDirectory=/opt/miden-network-monitor
```

Comment:
@bobbinth would we consider calling it just `miden-monitor`? Just for brevity's sake.

Reply:
I think this could be fine too, but I do think `miden-network-monitor` is a bit better as it is more descriptive. Not a strong preference though.